packages/formstr-sdk/src/utils/utils.ts

```diff
@@ -24,3 +26,13 @@ export function constructResponseUrl(privateKey: string) {
   }
   return `http://formstr.app/#/forms/${privateKey}/responses`;
 }
+
+export function detectFormVersion(form: FormSpec & V0FormSpec){
+  if(form.schemaVersion){
+    return form.schemaVersion
+  }
+  else if(form.schemaLink){
```

Comment:
should we give preference to schemaLink?

Reply:
we should, but will do that once we have examples of schemalinks working